gulp/util/karma-config.js

```diff
@@ -40,8 +40,8 @@ module.exports = function createConfig(project) {
         coverageReporter: {
             check: {
                 each: {
-                    lines: 79,
-                    statements: 79,
+                    lines: 78,
+                    statements: 78,
```

Comment:
might this be useful?
https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md